src/model/ircbuffermodel.cpp

```diff
@@ -605,6 +605,7 @@ void IrcBufferModelPrivate::_irc_restoreBuffers()
                     chans.clear();
                     keys.clear();
                     joinCommandLength = joinCommandMinLength;
+                    (void)joinCommandLength; // Fix a Static Analyzer warning that the variable is never read
```

Comment:
we probably shouldn't hide the issue but check if it's actually a bug that it's not used... @Venemo can you check this?